_includes/footer.html

```diff
@@ -1,12 +1,63 @@
-<footer class="usa-footer usa-footer-medium usa-section background-gray-dark" role="contentinfo">
+<footer class="usa-footer usa-section background-medium" role="contentinfo">
   <div class="usa-grid">
-    <h2>Drop us a line.</h2>
-    <p>To share your thoughts, email us anytime at <a href="mailto:{{ site.footer.email }}">{{ site.footer.email }}</a>.</p>
+    <div class="footer-logo-links">
+    {% include icons/oes.svg %}
```

Comment:
@mearley2013 @wslack you can actually keep the svg as is and just change the two color values in `_includes/icons/oes.svg`:

`<path fill="#1FD0C5" d="M0 48h48V0H0"/><g fill="#FFF">`

would be:
`<path fill="#FFF" d="M0 48h48V0H0"/><g fill="#205493">`

Since that's `_includes/icons/oes.svg` is being used for the logo, you could duplicate it.